src/widgets/table/table.rs

```diff
@@ -1484,7 +1364,264 @@ mod tests {
                 .rows(vec![])
                 .footer(Row::new(vec!["h", "i"]))
                 .column_spacing(0);
-            assert_eq!(table.get_columns_widths(10, 0), &[(0, 5), (5, 5)])
+            assert_eq!(table.get_columns_widths(10, 0), [(0, 5), (5, 5)])
+        }
+
+        #[test]
+        fn spacers_priority_unconstrained() {
+            fn test(
+                highlight_spacing: HighlightSpacing,
+                selection: Option<usize>,
+                expected: Buffer,
+            ) {
+                let table = Table::default()
+                    .rows(vec![Row::new(vec!["hello", "world"])])
```

Comment:
nit: it might be better to pick two words that don't tend to be presented as a single string, as it highlights that these are separate columns rather than a single common sentence. E.g. "ABCDE", "12345" might be more obvious.

Reply:
Changed to `ABCDE` and `12345`.